src/platform/user/profile/utilities/index.js

```diff
@@ -88,18 +88,22 @@ export function mapRawUserDataToState(json) {
       ? vet360ContactInformation
       : mockContactInformation,
     session,
+    veteranStatus: {
+      status: veteranStatus?.status,
+      isVeteran: veteranStatus?.isVeteran,
+      servedInMilitary: veteranStatus?.servedInMilitary,
+    },
   };
 
   if (meta && veteranStatus === null) {
     const errorStatus = meta.errors.find(
       error => error.externalService === commonServices.EMIS,
     ).status;
-    userState.veteranStatus = getErrorStatusDesc(errorStatus);
+    userState.veteranStatus.status = getErrorStatusDesc(errorStatus);
   } else {
-    userState.isVeteran = veteranStatus.isVeteran;
     userState.veteranStatus = {
       isVeteran: veteranStatus.isVeteran,
-      veteranStatus,
+      status: veteranStatus.status,
```

Comment:
We could just do `userState.veteranStatus: { ...veteranStatus }` (if I'm writing that object spread correctly)